app/services/id_token_builder.rb

```diff
@@ -55,6 +55,16 @@ def timestamp_claims
   end
 
   def acr
```

Comment:
High level comments:
1. I am not sure that this space-separate use of ACR is a good fit for the spec. Looking at the [iGov spec](https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#:~:text=OPTIONAL.%20The%20LoA%20the%20user%20was%20authenticated%20at.%20MUST%20be%20a%20member%20of%20the%20acr_values%20list%20from%20the%20authentication%20request.%20The%20OpenID%20Provider%20MUST%20NOT%20include%20this%20field%20if%20vot%20is%20provided.%20See%20Authentication%20Context%20for%20more%20details.), it says:
    > MUST be a member of the acr_values list from the authentication request

    which implies to me it's gotta be a singular value, and also references the request   
    Maybe is it time for us to look into the Vectors of Trust format to support these kinds of multiple values?
3. Can we add some tests for this?

Reply:
For 1, I followed the request parameters for the request response.

> Request Parameters
> acr_values
> The Authentication Context Class Reference requests can be used to specify the type of identity verification[1](https://developers.login.gov/oidc/#fn:1) or the AAL (Authentication Assurance Level) for the user. These and the scope determine which [user attributes](https://developers.login.gov/attributes/) will be available in the [user info response](https://developers.login.gov/oidc/#user-info-response).
> 
> Multiple values can be joined with a space (before being URI-escaped in the final URL)

Reply:
Yes but that's for request parameters, but this is a response object. I think it does not apply

Reply:
It is confusing, a request allows `acr_values`, but the response is only named `acr` 🤔

Perhaps we make up allowing returning `acr_values`?

Reply:
Yeah I would be a big fan of making a new claim/attribute. At that point it could even be an array instead of space-joined string